lib/inspec/plugin/v2/installer.rb

```diff
@@ -356,6 +356,15 @@ def install_gem_to_plugins_dir(new_plugin_dependency, # rubocop: disable Metrics
         # Skip in case using a gem based plugin
         next if spec.name == new_plugin_dependency.name && (update_mode || gem_based_plugin?(opts))
 
+        # give priority to already loaded gems
```

Comment:
Can you add some detailed comments over this block around it's functionality? Or the use cases this is handling? I know this might be old code.. but since we have got into it now..Will be good for future ref to understand code later.

Reply:
if somebody has to understand this code they have to dig into this either way .. but to make this process easier I shall add some comments